spot-setup/odm/README.md

```diff
@@ -0,0 +1,68 @@
+# spot-setup/odm - Open Data Model (ODM) Setup
+
+## Intended Audience
+
+This document is intended for any developer or sysadmin in learning the technical aspects or in contributing to the setup installation process of the Apache Spot solution.
+
+## Getting Started
+
+This information will help you to get started on contributing to the ODM component of the Apache Spot Setup repository.
+For information about installing and running Apache Spot go to our [Installation Guide](http://spot.apache.org/doc/).
+spot-setup/odm contains the scripts to setup HDFS for the Apache Spot Open Data Model (ODM).
+It will create the folder and database structure needed to run Apache Spot on HDFS and HIVE respectively.
+spot-setup/odm is a component of Apache Spot and is executed in the initial configuration after Linux user creation and before Ingest installation.
+
+## Prerequisites
+
+To collaborate and run spot-setup, it is required the following prerequisites:
+- A running Hadoop cluster
+- Linux user account created in all nodes with sudo privileges
+- HDFS ACLs must be enabled (dfs.namenode.acls.enabled=true)
+
+## General Description
+
+The main script in the repository is **odm_setup.sh** which is responsible of loading environment variables,
+creating folders in Hadoop for the ODM data (event, user_context, endpoint_context, network_context, threat_intelligence_context, vulnerability_context),
+create the Hive database,
+and finally execute hive query scripts that creates Hive tables needed to access Apache Spot data through the ODM.
+
+## Environment Variables
+
+**spot.conf** is the file storing the variables needed during the installation process including node assignment, User interface, Machine Learning and Ingest gateway nodes.
+This file also contains sources desired to be installed as part of Apache Spot, general paths for HDFS folders, Kerberos information and local paths in the Linux filesystem for the user as well as for machine learning, ipython, lda and ingest processes.
+To read more about these variables, please review the [wiki] (https://github.com/Open-Network-Insight/open-network-insight/wiki/Edit%20Solution%20Configuration).
```

Comment:
The ONI wiki, which may have some valuable information that should be pulled into the Spot docs or perhaps website, Is definitely out of date.

This is probably the best replacement for this link
http://spot.incubator.apache.org/doc/#configuration

Reply:
@natedogs911 no worries. The link has been updated.